weatherreport.cpp

```diff
@@ -36,6 +36,17 @@ namespace WeatherSpace
             return 52;
         }
     };
+	class ModifiedSensorStub : public SensorStub {
```

Comment:
Try reflecting the behavior of the Stub in its name. What if you make yet another stub, for a different weather pattern?